docker-compose.postgis.override.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  postgis_db_admin:
+    ports:
+      - ${postgis_pgadmin_port:-5052}:80
+    env_file:
```

Comment:
no need to redefine from default

Reply:
you mean the port? You prefer to break loudly? :)